plugins/leemons-plugin-learning-paths/frontend/src/components/ModuleDashboard/components/DashboardCard/components/DashboardCardBody/DashboardCardBody.js

```diff
@@ -43,7 +43,7 @@ const DashboardCardBody = ({ activity, statement, assetNumber }) => {
   useEffect(() => {
     getInstanceTypeLocale(activity);
   }, [activity]);
-  const subjectIds = [activity?.subjects?.[0]?.subject];
+  const subjectIds = [activity?.assignable?.subjects?.[0]?.subject];
```

Comment:
It should come from `useClassesSubjects(activity?.classes)`, because assignable subjects may contain more subjects than the assigned ones.